tests/lora/test_lora_layers_ltx_video.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2024 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import unittest
+
+import numpy as np
+import pytest
+import torch
+from transformers import AutoTokenizer, T5EncoderModel
+
+from diffusers import (
+    AutoencoderKLLTXVideo,
+    FlowMatchEulerDiscreteScheduler,
+    LTXPipeline,
+    LTXVideoTransformer3DModel,
+)
+from diffusers.utils.testing_utils import (
+    floats_tensor,
+    is_peft_available,
+    is_torch_version,
+    require_peft_backend,
+    skip_mps,
+    torch_device,
+)
+
+
+if is_peft_available():
+    pass
+
+sys.path.append(".")
+
+from utils import PeftLoraLoaderMixinTests, check_if_lora_correctly_set  # noqa: E402
+
+
+@require_peft_backend
+class LTXVideoLoRATests(unittest.TestCase, PeftLoraLoaderMixinTests):
+    pipeline_class = LTXPipeline
+    scheduler_cls = FlowMatchEulerDiscreteScheduler
+    scheduler_classes = [FlowMatchEulerDiscreteScheduler]
+    scheduler_kwargs = {}
+
+    transformer_kwargs = {
+        "in_channels": 8,
+        "out_channels": 8,
+        "patch_size": 1,
+        "patch_size_t": 1,
+        "num_attention_heads": 4,
+        "attention_head_dim": 8,
+        "cross_attention_dim": 32,
+        "num_layers": 1,
+        "caption_channels": 32,
+    }
+    transformer_cls = LTXVideoTransformer3DModel
+    vae_kwargs = {
+        "latent_channels": 8,
+        "block_out_channels": (8, 8, 8, 8),
+        "spatio_temporal_scaling": (True, True, False, False),
+        "layers_per_block": (1, 1, 1, 1, 1),
+        "patch_size": 1,
+        "patch_size_t": 1,
+        "encoder_causal": True,
+        "decoder_causal": False,
+    }
+    vae_cls = AutoencoderKLLTXVideo
+    tokenizer_cls, tokenizer_id = AutoTokenizer, "hf-internal-testing/tiny-random-t5"
+    text_encoder_cls, text_encoder_id = T5EncoderModel, "hf-internal-testing/tiny-random-t5"
+
+    text_encoder_target_modules = ["q", "k", "v", "o"]
```

Comment:
Fine to pass it off as there's no text encoder in the loadable components

Reply:
Yep. If we don't pass, there's a set of test failures. I think makes sense to pass whether or not we support finetuning it since those are the actual layer names in T5, and not q_proj, k_proj, etc.

```
FAILED tests/lora/test_lora_layers_ltx_video.py::LTXVideoLoRATests::test_get_adapters - ValueError: Target modules {'q_proj', 'out_proj', 'v_proj', 'k_proj'} not found in the base model. Please check the target modules and try again.
FAILED tests/lora/test_lora_layers_ltx_video.py::LTXVideoLoRATests::test_simple_inference_save_pretrained - ValueError: Target modules {'q_proj', 'out_proj', 'v_proj', 'k_proj'} not found in the base model. Please check the target modules and try again.
FAILED tests/lora/test_lora_layers_ltx_video.py::LTXVideoLoRATests::test_simple_inference_with_dora - ValueError: Target modules {'q_proj', 'out_proj', 'v_proj', 'k_proj'} not found in the base model. Please check the target modules and try again.
```